auth/src/main/res/drawable/ic_googleg_color_24dp.xml

```diff
@@ -5,15 +5,15 @@
     android:viewportHeight="512.0"
     android:viewportWidth="512.0">
     <path
-        android:fillColor="#4285f4"
+        android:fillColor="#ffffff"
```

Comment:
We should not modify Google logo. It's not correct in Zeplin, Google guidelines were updated